postprocessing.py

```diff
@@ -43,6 +43,46 @@ def transform(self, images):
         return {'categorized_images': categorized_images}
 
 
+class BuildingLabelerStream(BaseTransformer):
```

Comment:
@taraspiotr this class is dropped in the other PR as it is redundant so you can drop it here already